qiskit/providers/ibmq/jupyter/dashboard/job_widgets.py

```diff
@@ -73,54 +94,91 @@ def make_labels() -> widgets.HBox:
     return labels
 
 
+def make_rt_labels() -> widgets.HBox:
+    """Makes the labels widget.
+
+    Returns:
+        The labels widget.
+    """
+    labels0 = widgets.HTML(value="<h5>Job ID</h5>",
+                           layout=widgets.Layout(width='190px'))
+    labels1 = widgets.HTML(value='<h5>Program Name</h5>',
+                           layout=widgets.Layout(width='165px'))
+    labels2 = widgets.HTML(value='<h5>Status</h5>',
+                           layout=widgets.Layout(width='125px'))
+    labels3 = widgets.HTML(value='<h5>Created at</h5>',
+                           layout=widgets.Layout(width='100px'))
+
+    labels = widgets.HBox(children=[labels0, labels1, labels2, labels3],
+                          layout=widgets.Layout(width='700px',
+                                                margin='0px 0px 0px 35px'))
+    return labels
+
+
 def create_job_widget(watcher: 'IQXDashboard',
-                      job: IBMQJob,
+                      job: JobV1,
                       backend: str,
                       status: str = '',
-                      queue_pos: Optional[int] = None,
-                      est_start_time: Optional[datetime] = None) -> widgets.HBox:
+                      **kwargs) -> widgets.HBox:
```

Comment:
`backend` and `status` should both not be there because they can be derived, the others have to be parsed from `kwargs` because they are class-dependent